mllib/src/main/scala/org/apache/spark/ml/feature/HashingTF.scala

```diff
@@ -31,12 +31,9 @@ import org.apache.spark.sql.types.{ArrayType, StructType}
 /**
  * :: Experimental ::
  * Maps a sequence of terms to their term frequencies using the hashing trick.
- * Currently we support two hash algorithms: "murmur3" (default) and "native".
+ * Currently we support one hash algorithms "murmur3" which is also the default option.
```

Comment:
(Somewhat related to this PR) By the way, it is usual to select a vector size that is a power of 2, e.g. [scikit-learn doc mentions this](http://scikit-learn.org/stable/modules/feature_extraction.html#implementation-details):
> Since a simple modulo is used to transform the hash function to a column index, it is advisable to use a power of two as the n_features parameter; otherwise the features will not be mapped evenly to the columns.

We should add that to the doc